src/constants/constants.ts

```diff
@@ -0,0 +1,30 @@
+import { cardType } from "types";
+import { modalName } from "types/modalName.type";
+
+export const CARD_TYPES: Array<cardType> = [
+  { name: "포코", color: "gold" },
+  { name: "준", color: "#9edbd0" },
+  { name: "공원", color: "#55bf55" },
+  { name: "오스틴", color: "#ffa56f" },
+  { name: "후이", color: "#9198e5" },
+  { name: "유세지", color: "#d873ff" },
+  { name: "마르코", color: "#E76E9A" },
+  { name: "록바", color: "#d19c89" },
+];
+
+export const CARD_TYPES_DEFAULT: cardType = { name: "", color: "" };
+
+type modalNameConstant = { [key: string]: modalName };
+
```

Comment:
발견!!!!!!!!!! 이건 진짜 Record를 써야한다!!!!!!!!!!!!

```suggestion

type modalNameConstant =  Record<string, modalName >;

```

Reply:
인정!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

묘하게 다른 언어의 tuple 자료형이 생각나는 Record...!

Reply:
tuple..그런 무서운 말 하지마세요.